libpod/image/manifests.go

```diff
@@ -0,0 +1,146 @@
+package image
+
+import (
+	"context"
+
+	"github.com/containers/buildah/manifests"
+	"github.com/containers/image/v5/manifest"
+	"github.com/containers/image/v5/transports/alltransports"
+	"github.com/containers/image/v5/types"
+	"github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
+)
+
+// swagger:model ManifestAddOpts
+type ManifestAddOpts struct {
+	All        bool              `json:"all"`
+	Annotation map[string]string `json:"annotation"`
+	Arch       string            `json:"arch"`
+	Features   []string          `json:"features"`
+	Images     []string          `json:"images"`
+	OSVersion  string            `json:"os_version"`
+	Variant    string            `json:"variant"`
+}
+
+func (i *Image) InspectManifest() (*manifest.Schema2List, error) {
```

Comment:
Can we get comments on all the public functions in this file?

Reply:
Also - does this also work for OCI format images? A lot of this looks exclusive to Dockerv2

Reply:
I believe it does

Reply:
it really isnt contingent on an image ... the manifest list **is** the image.